src/GraphQL/Execution/ErrorInfoProvider.cs

```diff
@@ -65,14 +69,32 @@ public virtual ErrorInfo GetInfo(ExecutionError executionError)
                         extensions.Add("number", number);
                     if (data != null)
                         extensions.Add("data", data);
+                    if (stacktrace != null)
+                        extensions.Add("stacktrace", stacktrace);
```

Comment:
```suggestion
                        extensions.Add("stackTrace", stacktrace);
```
Names are camelCase, not lowercase, so I would think this should be "stackTrace"

Reply:
```suggestion
                        extensions.Add("details", stacktrace);
```
Or perhaps "details" as it will store the type names, messages and stack trace of outer and inner exceptions -- all information you'd want if enabling "stack traces".

Reply:
So... if that piece of information is some sort of _details_ then maybe just use `exception.ToString()` and not bother with manual building formatted string?

Reply:
I think that's what I suggested...  use "details" as the key and it equals `exception.ToString()`.

Reply:
OK but then `ExposeExceptionStackTrace` is not good enough name and something like `ExposeExceptionDetails`/`ExposeExceptionDetailsMode` suits better.

Reply:
StackTrace is just a StackTrace but we really have stack traces + exception types + exception messages

Reply:
Maybe just add `ExposeExceptionDetails` and deprecate the old property

Reply:
Or leave the old property as-is and add the new property...